frontend/test/components/measuretable.spec.jsx

```diff
@@ -80,6 +84,8 @@ describe('MeasureTable', () => {
       />
     );
 
-    expect(component).toMatchSnapshot();
+    expect(component).toMatchSnapshot({
+      lastUpdated: Date.now(),
+    });
```

Comment:
You can remove the arguments here, they should not be necessary (after you update the snapshot)